### PR TITLE
feat: 繁中服「照我以火」活動導航

### PR DESCRIPTION
**Pull request by momomochi987**
台服不知道為什麼左側變成顯示水月肉鴿的東西了，像這樣

Screenshot_20231019-162333

~~不能只偷懶改字就好 QQ~~

只好參考其他以圖片當辨識目標的 SideStory 寫法，寫一個照我以火的導航

如果還有需要調整的地方，再麻煩各位大佬了 ( ´•̥̥̥ω•̥̥̥` )

### CODE REVIEW

**Comment by Constrat** [comment]
Txwy resource failed ;-;

**Comment by momomochi987** [comment]
> Txwy resource failed ;-;
> 
> ```
> [2023-10-19 10:16:54.449][ERR][Px39c][Tx16a4] TaskData load failed, path:
> D:\a\MaaAssistantArknights\MaaAssistantArknights\x64\Debug\resource\global\txwy\resource\tasks.json ,
> templ dir:
> D:\a\MaaAssistantArknights\MaaAssistantArknights\x64\Debug\resource\global\txwy\resource\template
> ```

I  have no idea why it failed.... :(

It works perfectly on my local MAA

**Comment by Constrat** [comment]
@momomochi987 Give me a moment, I'll test on a debug version on my machine to make sure.

**Comment by ABA2396** [comment]
照理来说这种修改应该没啥问题，本地能跑就可以了

**Comment by ABA2396** [comment]
image
草）我本地怎么跑出来的还不一样

**Comment by ABA2396** [comment]
cc @MistEO

**Comment by ABA2396** [comment]
回家瞅瞅，感觉不像是这个提交的问题

**Comment by zzyyyl** [comment]
谁测了炸了的发下日志

**Comment by status102** [comment]
FC-Open原本是个OcrTask，之前有这种跨类型覆盖么

**Comment by ABA2396** [comment]
加个 "text_Doc": "skip" 试试

**Comment by Constrat** [comment]
But technically even if the name is the same. This should a create a standalone task from the CN

File: resource/global/txwy/resource/tasks.json [hunk 1]
```diff
@@ -4556,5 +4556,21 @@
             "尋訪十次",
             "訪十次"
         ]
+    },
+    "FC-Open": {
+        "algorithm": "MatchTemplate",
```
Comment (status102) [inline resource/global/txwy/resource/tasks.json:4561]:
```suggestion
        "bastTask": "#none",
        "algorithm": "MatchTemplate",
```

File: resource/global/txwy/resource/tasks.json [hunk 2]
```diff
@@ -4556,5 +4556,21 @@
...
+        "template": "FC-Open.png",
```
Comment (status102) [inline resource/global/txwy/resource/tasks.json:4562]:
```suggestion
        "baseTask": "#none",
        "algorithm": "MatchTemplate",
```

Comment (zzyyyl) [inline resource/global/txwy/resource/tasks.json:4562]:
```suggestion
        "baseTask": "#none",
```

Comment (zzyyyl) [reply resource/global/txwy/resource/tasks.json:4562]: this means this task does not inherit the CN task

**Comment by Constrat** [comment]
@momomochi987 MaaRelease?

**Comment by momomochi987** [comment]
啊啊昨天睡著了 QQ
感謝各位大佬幫忙 🙇‍♀️🙇‍♀️🙇‍♀️

**Comment by ABA2396** [comment]
> @momomochi987 MaaRelease?

The modification hot update involving pictures won't work

**Comment by Constrat** [comment]
I forgot that in this case pictures were needed, my bad

**Pull request merged**